spec/ddtrace/contrib/extensions_spec.rb

```diff
@@ -54,14 +54,19 @@
             end
           end
         end
+      end
 
+      describe '#configure_onto' do
+        include_context 'registry with integration' do
+          before { stub_const('Datadog::Contrib::REGISTRY', registry) }
+        end
         context 'given a target and options' do
-          subject(:configure) { described_class.configure(target, opts) }
+          subject(:configure_onto) { described_class.configure_onto(target, opts) }
 
           let(:target) { double('target') }
           let(:opts) { {} }
 
-          it { expect { configure }.to_not raise_error }
+          it { expect { configure_onto }.to_not raise_error }
         end
       end
     end
```

Comment:
Is this test still relevant? I'm guessing the intent previously was to check that the monkey patch did not break the core behavior, but now that the monkey patch does not touch the core behavior I think it may be time to just remove the test.

Reply:
Not relevant, thank you!